document/chapters/02-background.tex

```diff
@@ -929,14 +929,109 @@ \subsection{Signed Distance Fields}
+contributions to physical quantities at various locations in the physical
+model. Tallies on surfaces or in cells change the memory usage of the simulation
+insignificantly, unless additional data is needed to calculate derived
+quantities. Mesh-based tallies on the other hand can dominate the memory
+usage. In MCNP and many other Monte Carlo codes, mesh tallies are defined by the
+user as structured grids in either Cartesian, cylindrical, or spherical
+coordinates. These tallies are often used for better spatial resolution of
+physical data in the simulation and for coupling to analysis in other
+engineering domains. Tetrahedral mesh tallies are also supported by several
+Monte Carlo codes DAGMC interacts with, including MCNP
+\cite{LANL_MCNP5_VOLIII}. The size of these tallies vary based on the needs of
+the user and the trade-offs associated with spatial resolution of solutions and
+statistical convergence related to mesh element sizes.
+
+\subsection{Parallel Computation Schemes}
+
+For the Monte Carlo code used in this work, MCNP, the parallelism method applied
+is a master/slave scheme standard for many applications using the Message
+Passing Interface (MPI) \cite{Forum_1994}. The master process is responsible for
+initializing the problem, determining the load balance for the number of
+parallel processes requested by the user, and collecting/accumulating
+information at the end of the simulation. In MCNP, particle histories are
+divided into equal segments among the slave processes for evaluation. More
+information on the method for particle history division and random number
+sequence assignment was detailed by Deng and Xie \cite{Deng_1999}. For each
+slave process, a corresponding set of the nuclear data, tally data structures,
+and geometry representation are created. Duplication of the geometry
+representation has a significant implications for the use of CAD-based
+tessellations.
+
+\subsection{Impact on CAD-Base Geometry}
+
+The tessellations resulting from CAD geometries consume much more memory than
+corresponding CSG representations. This must be taken into consideration when
+planning parallel simulations clusters where the limitation for the number of
+processes per CPU can be limited by the memory used in a single process. Because
+DAGMC's CAD-based particle tracking is an addition to MCNP, it has no influence
+on the parallelism data model. Given that geometry representations are required
+to exist in each process of the parallel simulation, memory usage in DAGMC is
+closely monitored. Some of this memory usage is largely uncontrollable due to
+the reliance on CUBIT or Trelis' underlying tessellation algorithms,
+but additional data used to maintain geometric relationships and build acceleration data
+structures are kept to a minimum if possible.
+
+\begin{table}[H]
+    \centering
+  \begin{tabular}{l c c c}
+    \toprule
+    Model & Tessellation (MB) & Tessellation and MOAB BVH (MB) \\
```

Comment:
Can you add ITER to this?  These numbers aren't actually that large, but the ITER SDR model was many GB, as I recall.

Reply:
I thought about that, but at this point in the document I don't think I've introduced that model yet. I could introduce it here, though. It just seems like an awkward place to put a model description.

Reply:
Maybe exploring simulation memory usage would be helpful?